src/main/scala/com/lightbend/kafkalagexporter/ConsumerGroupCollector.scala

```diff
@@ -226,25 +243,57 @@ object ConsumerGroupCollector {
     /**
       * Refresh Lookup table.  Remove topic partitions that are no longer relevant and update tables with new Point's.
       */
-    private def refreshLookupTable(state: CollectorState, snapshot: OffsetsSnapshot, evictedTps: List[TopicPartition]): Unit = {
-      state.topicPartitionTables.clear(evictedTps)
-      for((tp, point) <- snapshot.latestOffsets) state.topicPartitionTables(tp).addPoint(point)
+    private def refreshLookupTable(log: Logger, topicPartitionTables: TopicPartitionTable, snapshot: OffsetsSnapshot, evictedTps: List[TopicPartition], redisClient: Option[RedisClient]): Unit = {
+      topicPartitionTables.clear(evictedTps)
+      for ((tp, point) <- snapshot.latestOffsets) {
+        topicPartitionTables(tp) match {
+          case Left(memory) =>
+            log.debug("  Point ({}, {}) was added in the in-memory lookup table ({}, {})", point.offset.toString, point.time.toString, tp.topic, tp.partition.toString)
+            memory.addPoint(point)
+          case Right(redis) =>
+            redisClient match {
+              case Some(client) =>
+                redis.addPoint(point, client) match {
```

Comment:
Once the table abstraction exists I suggest adding an `addPoint` overload, or some other method, to encapsulate adding a point and the associated logging.